js/CameraRoll.js

```diff
@@ -63,12 +57,47 @@ export type GetPhotosParams = {
    */
   assetType?: $Keys<typeof ASSET_TYPE_OPTIONS>,
 
+  /**
+   * Earliest time to get photos from. A timestamp in milliseconds. Exclusive.
+   */
+  fromTime?: number,
+
+  /**
+   * Latest time to get photos from. A timestamp in milliseconds. Inclusive.
+   */
+  toTime?: Number,
+};
+
+/**
+ * Shape of the param arg for the `getPhotos` function. This has a few more
+ * parameters than the `getPhotosFast` params, at the cost of some
+ * performance on iOS.
+ */
+export type GetPhotosParams = GetPhotosFastParams & {
+  /**
+   * A cursor that matches `page_info { end_cursor }` returned from a previous
+   * call to `getPhotos`
+   */
+  after?: string,
+
   /**
    * Filter by mimetype (e.g. image/jpeg).
    */
   mimeTypes?: Array<string>,
 };
 
+/**
+ * Params for the native `getPhotos` function, as implemented in the
+ * RNCCameraRoll module.
+ */
+export type GetPhotosNativeParams = GetPhotosParams & {
+  /**
+   * If provided, it's OK for the output to have empty filenames. This can
+   * improve performance on iOS when used by `getPhotosFast`.
+   */
+  allowEmptyFilenames?: boolean,
```

Comment:
This might be better named as something else: e.g., "skipGettingFilenames"

Reply:
Done!